README.md

```diff
@@ -30,7 +30,7 @@ walkdir = "1"
 The following code recursively iterates over the directory given and prints
 the path for each entry:
 
-```rust,no_run
+```no_run
```

Comment:
In the `README` I think we still need the `rust` annotation or GitHub won't do syntax highlighting.

Reply:
👍